staticcheck/doc.go

```diff
@@ -839,4 +839,27 @@ marshaling behavior, e.g. via MarshalJSON methods. It will also not
 flag empty structs.`,
 		Since: "2019.2",
 	},
+
+	"SA9006": {
+		Title: `Dubious bit shifting of a fixed size integer value`,
+		Text: `Bit shifting an value past its size will always clear the value.
+
+For instance:
+
+	v := int8(42)
+	v >>= 8
+
```

Comment:
Use 4 spaces instead of tabs for indentation, to be consistent with other examples.

Reply:
Fixed.